README.md

```diff
@@ -200,6 +205,27 @@ $preparedValues = $apiFilter->getPreparedValues($filters, $sql);
 // ]
 ```
 
+### `GT` + `LT` filter (_between_)
+```http request
+GET http://host/person/?age[gt]=18&age[lt]=30
+```
+
+```php
+$parameters = $request->query->all();
+// [
+//     "age" => [
+//         "gt" => 18
+//         "lt" => 30
+//     ],
+// ]
+
+$filters = $apiFilter->parseFilters($parameters);
+$sql = 'SELECT * FROM person';
+
+$sql = $apiFilter->applyFilters($filters, $sql); // SELECT * FROM person WHERE 1 AND age > :age_gt AND age < :age_lt
```

Comment:
Shouldn't be here `WHERE 1=1 AND` instead of `WHERE 1 AND`? 🤔

Reply:
It means the same. `1=1` will be calculated as `TRUE` which is `1` so there is `1` right away.